components/org.wso2.carbon.identity.application.authenticator.hypr.rest/org.wso2.carbon.identity.application.authenticator.hypr.rest.dispatcher/src/main/java/org/wso2/carbon/identity/application/authenticator/hypr/rest/dispatcher/ErrorConstants.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com). All Rights Reserved.
+ *
+ * This software is the property of WSO2 LLC. and its suppliers, if any.
+ * Dissemination of any information or reproduction of any material contained
+ * herein in any form is strictly forbidden, unless permitted by WSO2 expressly.
+ * You may not alter or remove any copyright or other notice from copies of this content.
+ *
+ */
+
+package org.wso2.carbon.identity.application.authenticator.hypr.rest.dispatcher;
+
+/**
+ * Error constants related to HYPR Authentication.
+ */
+public class ErrorConstants {
+
+    public static final String ERROR_CODE = "HYPR-API-";
```

Comment:
Same constant was defined in [DefaultExceptionMapper.java](https://github.com/wso2-extensions/identity-outbound-auth-hypr/pull/1/files#diff-8fb3439d122b04bf5c4fe6c92c1126abf4352ec952529abcc85fc8a03bb8521a). Is that possible to use one

Reply:
Fixed with [38a90ea](https://github.com/wso2-extensions/identity-outbound-auth-hypr/pull/1/commits/38a90eae1349a0988459ee5205d4d728aec92bca)